libcxx/include/tuple

```diff
@@ -220,7 +220,9 @@ template <class... Types>
 #  include <__config>
 #  include <__cstddef/size_t.h>
 #  include <__fwd/array.h>
+#  include <__fwd/complex.h>
 #  include <__fwd/pair.h>
+#  include <__fwd/subrange.h>
```

Comment:
It seems that modules builds need these inclusions to make `std::get` visible.

Reply:
How about including `<__fwd/get.h>` instead?

Reply:
I agree that seems overkill, but IMO that's better than including forward declarations for seemingly unrelated classes like `complex` and `subrange`. What's the modules error you got?